content/infrastructure/features.md

```diff
@@ -12,13 +12,13 @@ weight = 10
   icon = "boxes"
   icon_pack = "fas"
   name = "An environment for your community"
-  description = "2i2c Hubs provide data environments that have been designed for research and education, and can be customized for your community."
+  description = "2i2c managed JupyterHubs provide data environments that have been designed for research and education, and can be customized for your community."
 
 [[feature]]
   icon = "desktop"
   icon_pack = "fas"
   name = "Provide a variety of open source interfaces"
-  description = "Use a variety of user interfaces, including Jupyter Notebook, Jupyter Lab, RStudio, VSCode. If it can be run in a browser, it can run on a 2i2c Hub."
+  description = "Use a variety of user interfaces, including Jupyter Notebook, Jupyter Lab, RStudio, VSCode. If it can be run in a browser, it can run on a 2i2c managed JupyterHub."
```

Comment:
maybe rather than saying stuff like "run on a 2i2c managed JupyterHub", we instead say stuff like "run on your JupyterHub"?

Reply:
Yeah, I like that. Maybe 'your managed JupyterHub'? or is that redundant/

Reply:
I've rewritten the list of features so they don't need to refer to the 'hub'. I also removed 'content' and added 'authentication' to the list, since I think that's more important. LMK what you think